dialog/color.go

```diff
@@ -107,7 +110,10 @@ func (p *ColorPickerDialog) updateUI() {
 		w.Hide()
 	}
 	p.dialog.dismiss = &widget.Button{Text: lang.L("Cancel"), Icon: theme.CancelIcon(),
-		OnTapped: p.dialog.Hide,
+		OnTapped: func() {
+			p.callback(nil)
```

Comment:
I don’t think this part of the change is ok.
It will **panic** any apps which assume a colour chosen cannot be nil. Which is probably a lot of apps.

Reply:
OK. I will change. 
But this is all a bit inconsistent. The file dialogs explicitly work with such a nil argument.
And the confirmation dialog tests that the order of callbacks is the other way around.
For Fyne3 it would be great to get this straight.

I will go out for the rest of the day in a few minutes. So I will do the change tomorrow.

Reply:
Change is done.
All the best with your workshop at GoLab!